src/hooks/tokenomics/useTotalSupply.ts

```diff
@@ -0,0 +1,8 @@
+import getTotalSupply from 'api/tokenomics/getTotalSupply'
+import useSWR from 'swr'
+
+export default function useTotalSupply() {
+  return useSWR('tokenomics/totalSupply', async () => getTotalSupply(), {
+    refreshInterval: 60_000,
+  })
+}
```

Comment:
By using `useSWRImmutable` this hook will never update during the users session. It is unlikely, that the total supply ever updates, after it was fetched once.

```suggestion
import getTotalSupply from 'api/tokenomics/getTotalSupply'
import useSWRImmutable from 'swr/immutable'

export default function useTotalSupply() {
  return useSWRImmutable('tokenomics/totalSupply', async () => getTotalSupply())
}
```